src/index.html

```diff
@@ -183,6 +183,79 @@ <h2 class="h2">Next event</h2>
         </filter>
       </svg>
     </section>
+
+    <section class="bg-light-gray tablet:mx-4 my-16">
+      <div class="max-w-6xl mx-auto text-center px-4 py-20">
+        <h2 class="h2">Featuring talent from...</h2>
+  
+        <ul class="flex flex-col tablet:flex-row justify-between pb-11">
+          <li>
+            <img
+              class="mx-auto tablet:mx-0" 
+              src="./assets/svg/logo_automatic.svg" 
+              alt="Automattic logo"
+              width="224px"
+              height="68px"
+            >
+          </li>
+          <li>
+            <img 
+              class="mx-auto tablet:mx-0"
+              src="./assets/svg/logo_cocado.svg" 
+              alt="Cocado logo"
+              width="143px"
+              height="68px"
+            >
+          </li>
+          <li>
+            <img 
+              class="mx-auto tablet:mx-0"
+              src="./assets/svg/logo_keepcalling.svg" 
+              alt="Keep calling logo"
+              width="170px"
+              height="68px"
+            >
+          </li>
+          <li>
+            <img 
+              class="mx-auto tablet:mx-0"
+              src="./assets/svg/logo_total.svg" 
+              alt="Total energies logo"
+              width="60px"
+              height="68px"
+            >
+          </li>
+        </ul>
+  
+        <hr class="hr">
+  
+        <blockquote class="pt-11">
+          <img 
+            class="mx-auto"
+            src="./assets/svg/stars.svg" 
+            width="204px"
+            height="36px"
+            alt=""
```

Comment:
I think the alt text should be "Five stars" (I actually don't know if "**5** stars" or "**Five** stars" is better - maybe we can test with an actual screen reader some time to figure it out